async-nats/src/jetstream/stream.rs

```diff
@@ -331,6 +350,36 @@ pub struct StreamState {
     pub consumer_count: usize,
 }
 
+/// A raw stream message in the representation it is stored.
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct Message {
```

Comment:
Alternative name:

```suggestion
pub struct RawMessage {
```

We'll end up having `DirectMessage` too as that was just added to the server.